code/integration-tests/src/androidTest/java/com/adobe/marketing/mobile/integration/identity/IdentityIntegrationTests.kt

```diff
@@ -216,6 +216,74 @@ class IdentityIntegrationTests {
         countDownLatchSecondNetworkMonitor.await()
     }
 
+    @Test(timeout = 5_000)
+    fun testIdentitySendsForceSyncRequestOnLaunchOnValidConfig() {
+        SDKHelper.resetSDK()
+        ServiceProviderModifier.reset()
+
+        overrideNetworkService()
+
+        MobileCore.setApplication(ApplicationProvider.getApplicationContext())
```

Comment:
should we cover these in the setup instead?

Reply:
I refactored this test case. I originally thought I needed to restart the SDK, but I only needed to set the configuration and allow setUp to register the extensions for me.